src/content/meta/glossary.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Glossary
+description: A page with the term definitions Giant Swarm uses in the documentation.
+search: false
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - "What is a cloud-native developer platform?"
+  - "What is a developer environment?"
+  - "What is a management cluster?"
+  - "What is a managed app?"
+  - "What is the platform API?"
+  - "What is a platform operator?"
+  - "What is smart platform engineering?"
+  - "What is a workload cluster?"
+---
+
+We use a set of terms across our documentation that could be unclear to readers. In the following article we define those terms to clarify their meaning in the context of Giant Swarm.
+
+## Cloud native developer platform
+
+A cloud-native developer platform is an integrated collection of capabilities exposed through intuitive interfaces that facilitate the application development lifecycle. It offers flexible configuration to build a system that encodes your company practices. It integrates with different open source solutions to enable automation, connectivity, observability, and security for your workloads.
+
+## Developer environment
+
+In our context, a developer environment is the entity that defines the space where the developers will deploy their applications. It can be mapped to a Kubernetes namespace(s) or a cluster. The Platform capabilities like access management, security or observability are related to those environments.
+
+## Management cluster
+
+It is a Kubernetes cluster that acts a central point of the platform. It is in charge of hosting platform wide services like observability or security to ensure visualization and compliance over all the workloads in all clusters. [Platform operator](#platform-operator) have access to the its [API](#platform-api) to manage [workload clusters](#workload-cluster) and [managed apps](#managed-apps).
```

Comment:
```suggestion
A Management Cluster is a Kubernetes cluster that acts as a central management point of the platform. This cluster hosts platform wide services for gitops, observability, security and user management to ensure automation and visibility of the platform and workloads in all clusters. Users can create new clusters via a Management Cluster. [Platform engineers](#platform-engineers) usually have access to the [API](#platform-api) of the Management Cluster to manage [workload clusters](#workload-cluster) and services of the platform.
```

Reply:
As a user, I wouldn't understand

> automation and visibility of the platform and workloads in all clusters

What is made visible? What is automated? Maybe this can be a bullet point list. Very roughly like this:

- Users can create new clusters or apps through the API of the Management Cluster. In the same way, GitOps can be used to create and manage clusters and apps.
- Users can view installed apps and deployed workload clusters and their state through the API of the Management Cluster.
- The operators running on the Management Clusters manage the workload clusters and apps.
- (...something about observability and Grafana, maybe...)

Also `s/gitops/GitOps/`

Reply:
I have tried to simplify it. I would not define the whole concept here, it is just a glossary and IMO we can point to intro page for MC